### PR TITLE
Lyft 14491 more unit tests for secgroup

### PR DESCRIPTION
**Pull request by colinbjohnson**

### CODE REVIEW

**Comment by ryan-lane** [comment]
I think this change needs a rebase.

File: tests/unit/modules/boto_secgroup.py [hunk 1]
```diff
@@ -1,19 +1,69 @@
...
+        def stub_function(self):
+            pass
+        return stub_function
 
 # Import Salt Libs
 from salt.modules import boto_secgroup
 
 # Import Salt Testing Libs
-from salttesting import TestCase
+from salttesting import skipIf, TestCase
```
Comment (ryan-lane) [inline tests/unit/modules/boto_secgroup.py:34]: Are you still using skipIf?

Comment (colinbjohnson) [inline tests/unit/modules/boto_secgroup.py:34]: skipIf decorator is pretty useful - it fills the need of returning "skipped" and providing a skipped reason to the test runner - which the mock_ec2/stub_function don't do.

Comment (ryan-lane) [inline tests/unit/modules/boto_secgroup.py:34]: Ok. Cool. Just checking.

**Comment by colinbjohnson** [comment]
Closing - changes have been released upstream.

**Pull request closed without merging**